web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx

```diff
@@ -17,6 +17,7 @@ import FormLayout from 'layouts/FormLayout';
 
 import React from 'react';
 import { Trans } from '@lingui/macro';
+import { BiDetail } from 'react-icons/bi';
```

Comment:
What do you think if we use the eyes icon instead?

Reply:
That's fine. I'll make the change.

Reply:
@masskoder The icon has been updated. A screenshot with the eye icon is below.

![TRISA Directory Service (9)](https://github.com/trisacrypto/directory/assets/94616884/64004107-b6f4-47a3-93b1-4842183376d4)